IRIS-4148 Make sure lazy loaded images appear in discussions

garthwebb opened the pull request:
Our approach removes the lazy loading aspects and makes these normal images.

Ticket:

https://wikia-inc.atlassian.net/browse/IRIS-4148

Commit:
IRIS-4148 First version of de-lazifying.  Could use some improvements

Commit:
IRIS-4148 Simplify the fix.  Just use the noscript version of the image that is already in the content

garthwebb commented on extensions/wikia/Discussions/maintenance/ForumDumper.php:
```diff
@@ -111,6 +111,13 @@ public function getPages() {
+						$row->$prop = 0;
+					}
+				}
+
```
This code eliminates some noisy warnings.

mszabo-wikia replied:
AFAIK we performed a site-wide schema update in  [SUS-1690](https://wikia-inc.atlassian.net/browse/SUS-1690), so these columns should no longer appear anywhere (and they were never used)

garthwebb replied:
@TK-999, thanks for the update!  I've seen some still in DEV so its good to know I don't need to worry about them in production.

Commit:
IRIS-4148 Send plaintext back for rawContent rather than wikiText

Its more useful for us to have plaintext in discussions given that discussoins can't do anything with wikitext

jsutterfield commented:
LGTM
